### DISCUSSION TITLE
How to Build a Ticketing System on Chatwoot?

### DISCUSSION THREAD

**Discussion by Ladet02** [chatwoot/chatwoot#8022] [Q&A]
Hi, I have a python django app. I'm currently using the chat and setUser js to get customer details. But I want to build a custom user interface  with the following features:

- Users are able to Create a ticket (I think chatwoot refers to this as conversations)
- Users can see all their previous tickets  
- This is all in sync with the chat widget.

Also additionally, in the chat widget how do I build popular articles, similar to what's on chatwoots website widget? so users can read FAQ before starting an actual conversation see bellow for screenshot.

![image](https://github.com/chatwoot/chatwoot/assets/75883669/c2ee71ba-a994-47f8-b465-75255bf2fea6)

**Comment by Ladet02** [comment]
@pranavrajs any help?

**Comment by kry** [comment]
I would also like to integrate the popular articles in the chat window.
Is this available somewhere to enable?
